src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -27,24 +34,47 @@ public class PerformerManagementController : Base.MCController
         private const int KitsPerPage = 15;
         private const int PerformersPerPage = 15;
 
+        private const string ExcelMimeType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
+        private const string ExcelFileExtension = "xlsx";
+        private const int ExcelStyleIndexBold = 1;
+        private const int ExcelPaddingCharacters = 1;
+
+        private readonly IEnumerable<object> headerRow = new object[]
+        {
+            "Todays Date",
+            "Performer Namer",
+            "Billing Month",
+            "Vendor Id",
+            "Performer Description",
+            "Total Cost",
+            "Performer Billing"
+        };
+
         private static readonly DateTime DefaultPerformerScheduleStartTime = DateTime.Parse("8:00 AM");
         private static readonly DateTime DefaultPerformerScheduleEndTime = DateTime.Parse("8:00 PM");
 
         private readonly ILogger<PerformerManagementController> _logger;
         private readonly PerformerSchedulingService _performerSchedulingService;
         private readonly SiteService _siteService;
+        private readonly Domain.Repository.IBranchRepository _branchRepository;
```

Comment:
Repositories shouldn't be referenced in the controller, SiteService should have the needed branch calls